src/library_openal.js

```diff
@@ -1765,6 +1766,7 @@ var LibraryOpenAL = {
     };
     var onSuccess = function(mediaStream) {
       newCapture.mediaStreamSourceNode = newCapture.audioCtx.createMediaStreamSource(mediaStream);
+      newCapture.mediaStreamTrack = mediaStream;
```

Comment:
it's surprising this does `.mediaStreamTrack = mediaStream`, that is, why is there "track" on one side but not both. I don't know this API that well though, is the reason obvious? Or should this be `.mediaStream = mediaStream`?

Reply:
I referred to this.

Reply:
I don't know what you mean by "on one side but not both," but I can rename mediaStreamTrack to mediaStream.

Reply:
Sorry, I meant that the word "track" was on one side (`newCapture.mediaStreamTrack`) and not the other which had the value to be assigned.